source/frontend/public/tenants/tenant.json

```diff
@@ -48,14 +48,5 @@
   "idlePromptTimeout": 15,
   "idleTimeout": 15,
   "doubleClickInterval": 250,
-  "telemetry": {
-    "enabled": true,
-    "debug": false,
-    "environment": "local",
-    "serviceName": "frontend",
-    "endpoint": "https://collector-3cd915-dev.apps.silver.devops.gov.bc.ca",
-    "exportInterval": 10000,
-    "histogramBuckets": [0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10]
-  },
```

Comment:
I had to move the telemetry config out from the tenant.json file because I noticed issues with the telemetry setup running multiple times - the first render will use the "defaultTenant" and then when tenant.json is fetched via XHR then the context is updated and a re-render happens.

This was messing up the telemetry code so I moved it to a different pattern of injecting config values, which is:
- In development (ie **npm start** in our locals) it uses the .env files to load the values
- In production (ie **npm run build**) it relies on a **config.js** file that is loaded in index.html. The file is empty by default on our source code but it gets overridden with a config-map in Openshift environments.

See: https://www.justinpolidori.it/posts/20210913_containerize_fe_react/